README.md

```diff
@@ -137,6 +137,8 @@ bear keeping in mind:
 * By default, `fetch` **won't send any cookies** to the server, resulting in
   unauthenticated requests if the site relies on maintaining a user session.
 
+* By default, the **browser will not store any cookies** set in `fetch()` responses.
```

Comment:
Since the two bullet points are closely related, could we collapse them into one, explaining how the browser will not handle cookies at all, either sending or receiving?